src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -260,14 +247,9 @@ protected function setupAdmin()
         $userManager = $this->getContainer()->get('fos_user.user_manager');
         $user = $userManager->createUser();
 
-        $question = new Question('Username (default: wallabag) :', 'wallabag');
-        $user->setUsername($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
-
-        $question = new Question('Password (default: wallabag) :', 'wallabag');
-        $user->setPlainPassword($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
-
-        $question = new Question('Email:', '');
-        $user->setEmail($questionHelper->ask($this->defaultInput, $this->defaultOutput, $question));
+        $user->setUsername($this->io->ask('Username', 'wallabag'));
+        $user->setPlainPassword($this->io->askHidden('Password'));
```

Comment:
Warning, this could be a potential BC break : when using this method, password can't have default value.

Let me know if you want me to revert this change.